repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -261,6 +323,12 @@ class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMo
   def getOutputStreamAsInputStream: InputStream = bodyOutputStream.getOutputStreamAsInputStream
 
   /**
+    * @throws IllegalStateException when bodyMode is ResponseMode.PASSTHROUGH
+    */
+  def getOutputStreamAsString: String =
```

Comment:
It's just a convenience method. I was often getting the output stream as an input stream to then convert it to a string. It came up a lot in testing, but I can see it being useful in production code as well. I don't know that it has been used too much yet since I added it fairly late.